### PR TITLE
Nini looser

### PR DESCRIPTION
**Pull request by fdamhaut**
@client.command(name="nini")
async def niniVainz(ctx):
    if ctx.author.id == 291882226927271936:
        utils.send_msg("MDR, T KI?", ctx)

(Code de Vainzou)

Il faut update settings pour mettre "loooooser.euses" comme loser_role_name

Voilà Voilà, dit moi quoi

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -144,62 +145,145 @@ async def nini(ctx):
...
+        res.append("`{} | {} | {} | {} | {} | {} | {}`".format(username.center(20), str(data["messages"]).center(12), str(data["errors"]).center(12),
+                                                          str(int(data["messages"]/data["errors"])).center(7), str(data["streak"]).center(12), str(data["streak_max"]).center(12), data["rank_update"].center(12)))
 
     log_info('\n'.join(res))
-    res.append("J'ai lu {} messages !".format(c))
+
+
+    # LOSER MESSAGE AND ROLES
+
+    nLoser = max(NUMBER_OF_LOSER, len(loss_ranking))
```
Comment (Serpod) [inline main.py]: If there is no loser, nLoser will be equal to NUMBER_OF_LOSER which is 2.

What's the purpose of this?

File: main.py [hunk 2]
```diff
@@ -144,62 +145,145 @@ async def nini(ctx):
...
+
+    # REMOVE ALL CURRENT LOSERS
+    role = get(ctx.guild.roles, name=LOSER_ROLE_NAME)
+
+    for member in ctx.guild.members:
+        if role in member.roles:
+            await member.remove_roles(role)
+
+    for name, r in loss_ranking:
+        if name in all_losers and "id" in all_losers[name]:
```
Comment (Serpod) [inline main.py:280]: If someone is in `loss_ranking`, they have to already be in all_losers. Is this line useful? Is it some kind of safeguard?

File: main.py (hunk 1 above)
Comment (fdamhaut) [reply main.py]: Should be min instead of max

File: main.py (hunk 2 above)
Comment (fdamhaut) [reply main.py:280]: safeguard, i know try/catch is more efficient than that defensive programming but force of habit

Comment (fdamhaut) [reply main.py:280]: than defensive*

File: main.py (hunk 1 above)
Comment (fdamhaut) [reply main.py]: Alsor resolved L206-208 which forgot to remove old user without new loss

**Pull request merged**